src/update.ts

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+import * as uuid from 'uuid';
+import { bufferToHex } from 'ethereumjs-util'
+import { helpers } from 'leap-core';
+import Web3 from 'web3';
+
+import dynamoDb from './dynamodb'
+
+export const updateStats = async (_event, _context, callback) => {
+  const plasmaProvider = 'wss://testnet-node1.leapdao.org:1443';
+  const web3Plasma = helpers.extendWeb3(new Web3(plasmaProvider));
+
+  await web3Plasma.eth.net.getId();
+
+  const uTxos = await web3Plasma.getUnspent();
+
+  const unspents: import('./types').UnspentWithTx[] = await Promise.all(
+    uTxos.map((u: import('./types').UnspentWithTx) =>
+      web3Plasma.eth
+        .getTransaction(bufferToHex(u.outpoint.hash))
```

Comment:
would this block work with `await` here instead of chaining? To keep the consistency in style? :)

Reply:
Good idea. I'll update this. Thanks for pointing out 👍